src/seedu/addressbook/AddressBook.java

```diff
@@ -211,13 +211,17 @@ public static void main(String[] args) {
         processProgramArgs(args);
         loadDataFromStorage();
         while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
+            processUserCommand();
         }
     }
 
+	private static void processUserCommand() {
```

Comment:
Good idea to extract the method. However, the header comments are missing for the new method. All non trivial methods should have javadoc format header comments.